pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTStage.java

```diff
@@ -281,24 +170,24 @@ public ModelASTStages getParallel() {
         return parallel;
     }
 
-    public List<ModelASTParallelContent> getParallelContent() {
+    @Deprecated
+    public void setParallel(ModelASTStages s) {
```

Comment:
eehh? :confused:

Reply:
I swear there was a reason for this...probably demented worrying about in-progress-runs during upgrade. Which I think we can disregard, since I seem to have decided that we don't guarantee in-progress runs will work when upgrading a minor version (i.e., we broke that for the 1.1.x to 1.2.x upgrades, so we can do it for 1.2.x to 1.3.x, especially since this would be far less likely to cause problems than the 1.1.x->1.2.x upgrade and I never heard any complaints about that not working anyway).